portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,22 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> test;
 
+  @Override
+  public void init() { 
+    test = new ArrayList<>();
+    test.add("I love you.");
+    test.add("How are you?");
+    test.add("My name is David.");
+    
+  }
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello David!</h1>");
+    // JSON json = JSON.parse(test);
```

Comment:
Try to get in the habit of not checking in commented out bits of code since this will help for when you start collaborating with others more – it'll likely be unclear to a collaborator / reader if this code is useful or what it's context is (should it be uncommented at some point? is it outdated?, etc)

Reply:
will do